src/clr-angular/emphasis/alert/alert.ts

```diff
@@ -23,13 +23,13 @@ import { AlertIconAndTypesService } from './providers/icon-and-types.service';
 import { MultiAlertService } from './providers/multi-alert.service';
 import { isBooleanAttributeSet } from '../../utils/component/is-boolean-attribute-set';
 import { ClrCommonStringsService } from '../../utils/i18n/common-strings.service';
-import { AriaLiveService, AriaLivePoliteness } from '../../utils/a11y/aria-live.service';
+import { ClrAriaLiveService, ClrAriaLivePoliteness } from '../../utils/a11y/aria-live.service';
 import { ClrAlertText } from './alert-text';
 import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'clr-alert',
-  providers: [AlertIconAndTypesService, AriaLiveService],
+  providers: [AlertIconAndTypesService, ClrAriaLiveService],
```

Comment:
We should remove it from all providers here, since it should be provided in root.